Sources/Plasma/PubUtilLib/plNetMessage/plNetMessage.h

```diff
@@ -45,7 +45,6 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "HeadSpin.h"
 #include "hsBitVector.h"
 
-#include "pnNetBase/pnNetBase.h"
```

Comment:
I think this means you can remove the pnNetBase `target_link_library` from CMakeLists.txt